firebase-auth/src/commonMain/kotlin/dev/gitlive/firebase/auth/credentials.kt

```diff
@@ -15,6 +15,7 @@ expect class OAuthCredential : AuthCredential
 
 expect object EmailAuthProvider {
     fun credential(email: String, password: String): AuthCredential
+    fun getCredentialWithLink(email: String, emailLink: String): AuthCredential
```

Comment:
can we rename to credentialWithLink for consistency

Reply:
Yes, we can. Thanks for the comment.